### PR TITLE
docs: add kapt parameters

### PR DESCRIPTION
**Pull request by p7nov**

### CODE REVIEW

File: pages/docs/reference/kapt.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ kapt {
 
 </div>
 
-## Gradle Build Cache Support (since 1.2.20)
+## Gradle build cache support (since 1.2.20)
```
Comment (alexey-tsvetkov) [inline pages/docs/reference/kapt.md]: I think that "Gradle Build Cache" is kind of a proper noun, so all words should be capitalized

Comment (p7nov) [reply pages/docs/reference/kapt.md]: In [Gradle docs](https://docs.gradle.org/current/userguide/build_cache.html), it's capitalized only in headings. We use a different heading capitalization style: Gradle uses the traditional `Title Case` and we prefer `sentence case` as does [Microsoft](https://docs.microsoft.com/en-us/style-guide/capitalization), for example.
In sentences, Gradle spells it `build cache` (not a proper noun), so I'd stick to this spelling in both headings and sentences.

File: pages/docs/reference/kapt.md [hunk 2]
```diff
@@ -110,7 +110,44 @@ kapt {
...
-## Incremental Annotation Processing (since 1.3.30)
+## Running kapt tasks in parallel (since 1.2.60)
+
+To improve the speed of builds that use kapt, you can enable the [Gradle worker API](https://guides.gradle.org/using-the-worker-api/) for kapt tasks.
+Using the worker API lets Gradle run independent annotation processing tasks from a single project in parallel, which in some cases significantly decreases the execution time. 
+However, running kapt with Gradle worker API enabled can result in increased memory consumption due to parallel execution. 
+
+To use the Gradle worker API for parallel execution of kapt tasks, add this line to your `gradle.properties` file:
+
+<div class="sample" markdown="1" mode="xml" theme="idea">
```
Comment (AlexanderPrendota) [inline pages/docs/reference/kapt.md]: really xml?

File: pages/docs/reference/kapt.md [hunk 3]
```diff
@@ -110,7 +110,44 @@ kapt {
...
+* The project's source files are unchanged.
+* The changes in dependencies are [ABI](https://en.wikipedia.org/wiki/Application_binary_interface) compatible. For example, the only changes are in method bodies. 
+
+However, compile avoidance can't be used for annotation processors discovered in the compile classpath since _any changes_ in them require running the annotation processing tasks. 
+
+To run kapt with compile avoidance:
+* Add the annotation processor dependencies to the `kapt*` configurations manually as described [above](#using-in-gradle).
+* Turn off the discovery of annotation processors in the compile classpath by adding this line to your `gradle.properties` file:
+
+<div class="sample" markdown="1" mode="xml" theme="idea">
```
Comment (AlexanderPrendota) [inline pages/docs/reference/kapt.md]: rly xml?

**Pull request merged**